Add `MWC 7K 2024` news post, wiki article (#10646)
ppy/osu-wiki: wiki/Tournaments/MWC/2024_7K/en.md

---
tags:
  - MWC2024
  - MWC 2024
  - MWC7K 2024
  - MWC 7K 2024
---

# osu!mania 7K World Cup 2024

![MWC 7K 2024 banner](img/mwc7k2024-banner.jpg)

The **osu!mania 7K World Cup 2024** (***MWC 7K 2024***) is a country-based osu!mania tournament hosted by the [osu! team](/wiki/People/osu!_team). It is the eighth instalment of the osu!mania 7K World Cup.

## Tournament schedule

| Event | Timestamp |
| --: | :-- |
| Registration phase | 2023-12-15 (00:00 UTC)/2023-12-28 (23:59 UTC) |
| Qualifier showcase | 2024-01-07 (14:00 UTC) |
| Qualifier stage | 2024-01-13/2024-01-14 |
| Round of 24 | 2024-01-20/2024-01-21 |
| Round of 16 | 2024-01-27/2024-01-28 |
| Quarterfinals | 2024-02-03/2024-02-04 |
| Semifinals | 2024-02-10/2024-02-11 |
| Finals | 2024-02-17/2024-02-18 |
| Grand Finals | 2024-02-24/2024-02-25 |

## Prizes

> The osu!mania 7K World Cup 2024 offers a minimum $1,200 cash prize pool and limited edition merch.

Reviewer: ```suggestion
The osu!mania 7K World Cup 2024 offers a minimum $1,200 cash prize pool and limited-edition merch.
```

4. The mappool is to be played according to the order listed on this page.
   - Teams may skip playing any beatmap on their second run.
5. Each team must have 2 members play each map. They may be exchanged freely after a map is concluded.
6. An optional 3-minute break will be offered between the first and second playthrough of the mappool.
7. All teams will play their qualifiers in separate rooms. It is *suggested* that teams do not broadcast or share their results publicly to avoid seed manipulation.
   - Multiple teams may be grouped into a singular room if there are not enough referees available at match time.
8. Only the top 24 seeded teams will advance to the Round of 24.

### Stage instructions

1. Following the Qualifiers, a double-elimination bracket will be played — for teams in the winners bracket, the winning team continues in the same bracket, while the losing team gets moved to the losers bracket, where any further defeats will eliminate the team from the competition.
2. Teams in the losers bracket must play up to 2 matches each weekend, starting from the Semifinals.

> 3. The Top 8 seeded teams from the Qualifiers get a bye for the Round of 24, and start playing on the Round of 16 instead.

Reviewer: ```suggestion
3. The top 8 seeded teams from the Qualifiers get a bye for the Round of 24, and start playing on the Round of 16 instead.
```

2. Each mappool consists of a fixed amount of maps, which will all be played under Free Mod conditions. 
3. The mappool sizes are as follows:
   - Qualifiers: 8 beatmaps
   - Round of 24 and Round of 16: 12 beatmaps
   - Quarterfinals and Semifinals: 14 beatmaps
   - Finals and Grand Finals: 16 beatmaps
4. Each mappool has one tiebreaker, except for the Qualifiers.
5. Possible mod choices for all beatmaps are [Hidden](/wiki/Gameplay/Game_modifier/Hidden), [Fade In](/wiki/Gameplay/Game_modifier/Fade_In), [Flashlight](/wiki/Gameplay/Game_modifier/Flashlight), and [Mirror](/wiki/Gameplay/Game_modifier/Mirror).
6. Failed scores will be counted in their integrity. To that end, the [No Fail](/wiki/Gameplay/Game_modifier/No_Fail) mod will be enforced on every beatmap.
   - Teams that purposefully disregard this instruction and team fail as a result will lose said game.
7. The tiebreaker will be played under Free Mod conditions.

### Scheduling information

1. Each stage will be held on **a single weekend**.
2. Matches in Qualifiers will be held during specific time slots proposed by the tournament managers. Each team may to choose a time slot in which they want to play.

>    - **A team that doesn't pick a time slot by January 10, 23:59 UTC will be considered disqualified.**

Reviewer: ```suggestion
   - **A team that does not pick a time slot by January 10, 23:59 UTC will be considered disqualified.**
```
